src/help/components/landing/IllustratedListTile.tsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable uniprot-website/use-config-location */
+import { ReactElement } from 'react';
+import { Link } from 'react-router-dom';
+import cn from 'classnames';
+import { RequireExactlyOne } from 'type-fest';
+import { LocationDescriptor } from 'history';
+
+import styles from './styles/illustrated-list-tile.module.scss';
+
+type LinkOrHref = RequireExactlyOne<
+  {
+    label: string;
+    to: LocationDescriptor<unknown>;
```

Comment:
```suggestion
    to: LocationDescriptor;
```
Don't think the `unknown` generics adds much